docs/installation/cloud/gcp.rst

```diff
@@ -17,42 +30,242 @@ To deploy VyOS on GCP (Google Cloud Platform)
+2. Select select a project.
 
-.. figure:: /_static/images/cloud-gcp-03.png
+.. figure:: /_static/images/cloud-gcp-proj.png
 
-4. After few seconds click to ``instance``
+3. Click **Create VPC Network**.
 
-.. figure:: /_static/images/cloud-gcp-04.png
+   **Public VPC**:
+
+   - Name: e.g., ``vyos-public-vpc``
+
+   - Subnet creation mode: ``Custom``
+
+   - Subnet name: e.g., ``vyos-public-subnet``
+
+   - Region: e.g., ``europe-west-1``
+
+   - IP range: e.g., ``10.0.1.0/24``
+
+   - Leave all other settings at default, then click **Create**.
+
+.. figure:: /_static/images/cloud-gcp-vpc-01.png
+
+.. figure:: /_static/images/cloud-gcp-vpc-02.png
+   
+   **Private VPC**:
+
+   - Name: ``vyos-private-vpc``
+
+   - Subnet creation mode: ``Custom``
+
+   - Subnet name: ``vyos-private-subnet``
+   
+   - Region: e.g., ``europe-west-1``
+
+   - IP range: ``10.0.11.0/24``
+
+   - Leave all other settings at default, then click **Create**.
+
+.. figure:: /_static/images/cloud-gcp-vpc-03.png
+
+.. figure:: /_static/images/cloud-gcp-vpc-04.png
+
+4. Add firewall rules to allow specific network traffic from the Internet. By default all incoming traffic from outside a network is blocked.
+
+.. figure:: /_static/images/cloud-gcp-vpc-05.png
+
+.. figure:: /_static/images/cloud-gcp-vpc-06.png
+
+.. figure:: /_static/images/cloud-gcp-vpc-07.png
+
+For more information, please visit the official Google Cloud documentation:
+
+https://cloud.google.com/vpc/docs/create-modify-vpc-networks
+
+
+Step 4: Deploy VyOS instance from Marketplace
+---------
 
-5. Find out your external IP address
+1. Go to the Google Cloud Marketplace page in the Google Cloud console  https://console.cloud.google.com/marketplace
 
-.. figure:: /_static/images/cloud-gcp-05.png
+2. Select select a project.
 
-6. Connect to the instance. SSH key was generated in the first step.
+.. figure:: /_static/images/cloud-gcp-proj.png
+
+3. On Google Cloud Marketplace search "vyod"
+
+.. figure:: /_static/images/cloud-gcp-market-01.png
+
+.. figure:: /_static/images/cloud-gcp-market-02.png
+
+4. On the next page, you can find information about `support`, `pricing`, and other details.
+
+.. figure:: /_static/images/cloud-gcp-market-03.png
+
+5. Click ``GET STARTED`` button to start deployment process.
+
+.. figure:: /_static/images/cloud-gcp-market-04.png
+
+.. figure:: /_static/images/cloud-gcp-market-05.png
+
+6. General settings.   
+
+   - Deployment name: e.g., ``vyos-test-vm``
+
+   - Select a Service Account: Select a Service Account which you created before.
+
+   - Image: Select VyOS image for deployment.
+
+   - Zone: e.g., ``europe-west-1-b``
+
+.. figure:: /_static/images/cloud-gcp-vm-01.png
+   
+.. figure:: /_static/images/cloud-gcp-vm-02.png
+
+7. Networking.
+
+   - Public Network interface:
+
+   - Network: ``vyos-public-vpc``
+
+   - Subnetwork: ``vyos-public-subnet``
+
+   - External IP: ``Ephemeral``
+
+   - Private Network interface:
+
+   - ADD A NETWORK INTERFACE.
```

Comment:
<img width="335" alt="Bildschirmfoto 2025-04-22 um 21 27 15" src="https://github.com/user-attachments/assets/e38b8da6-c292-409c-a76d-145b5634b7e7" />
this look strange.